homework16/SpringCoreProject/src/main/java/ua/lviv/home/SpringCoreProject/dao/StudentDao.java

```diff
@@ -0,0 +1,44 @@
+package ua.lviv.home.SpringCoreProject.dao;
+
+import ua.lviv.home.SpringCoreProject.entity.Student;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class StudentDao implements CRUD {
+
+    List<Student> students = new ArrayList<>();
+
+    @Override
+    public Student create(Student student) {
+        student.setId(students.size());
+        students.add(student);
+        return student;
+    }
+
+    @Override
+    public Student readById(int id) {
+        Student student = students.get(id - 1);
```

Comment:
In one of the students will be removed, we won't be able to get the last students.

For example, we have 3 students with ids 1, 2, 3.
We remove the student with id 2.
Then when we will try to get a student with id 3 we will get the following code 
```
students.get(2)
```
But we have only 2 elements in the list so we will get `IndexOfBoundExeption`.

Reply:
changed the implementation of methods for the correct operation of the project